src/main/java/org/ays/emergency_application/model/request/EmergencyEvacuationApplicationRequest.java

```diff
@@ -115,9 +116,9 @@ private boolean isPhoneNumberMustNotBeSameOne() {
     @SuppressWarnings("This method is unused by the application directly but Spring is using it in the background.")
     private boolean isSourceCityAndDistrictDifferentFromTargetCityAndDistrict() {
 
-       if (this.sourceCity == null || this.sourceDistrict == null || this.targetCity == null || this.targetDistrict == null) {
+        if (Strings.isNullOrEmpty(this.sourceCity) || Strings.isNullOrEmpty(this.sourceDistrict) || Strings.isNullOrEmpty(this.targetCity) || Strings.isNullOrEmpty(this.targetDistrict)) {
```

Comment:
`org.apache.commons.lang3.StringUtils` sınıfı kullanılabilir;
```suggestion
        if (StringUtils.isBlank(this.sourceCity) || StringUtils.isBlank(this.sourceDistrict) || StringUtils.isBlank(this.targetCity) || StringUtils.isBlank(this.targetDistrict)) {
```

Reply:
Değiştirdim